presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTableStatistics.java

```diff
@@ -758,6 +759,31 @@ public void testAnalyzeForTablePartitionedByBigint()
                 row(null, null, null, null, 5.0, null, null));
     }
 
+    @Test
+    @Requires({NationPartitionedByBigintTable.class, UnpartitionedNationTable.class})
+    public void testEstBytesScannedAcrossJoinedTables()
+    {
+        String tableName1InDatabase = mutableTablesState().get(NATION.getName()).getNameInDatabase();
+        String tableName2InDatabase = mutableTablesState().get(NATION_PARTITIONED_BY_BIGINT_REGIONKEY.getName()).getNameInDatabase();
+
+        assertThat(query("ANALYZE " + tableName1InDatabase)).containsExactly(row(25));
+        assertThat(query("ANALYZE " + tableName2InDatabase)).containsExactly(row(15));
+
+        assertThat(query("SHOW STATS FOR " + tableName1InDatabase)).containsOnly(
+                row("n_nationkey", null, 25.0, 0.0, null, "0", "24"),
+                row("n_name", 177.0, 25.0, 0.0, null, null, null),
+                row("n_regionkey", null, 5.0, 0.0, null, "0", "4"),
+                row("n_comment", 1857.0, 25.0, 0.0, null, null, null),
+                row(null, null, null, null, 25.0, null, null));
```

Comment:
what changed here? How this assertion would look like before your change in the product?